### PR TITLE
Fix rocksmq produce hang when do search repeatedly

### PR DESCRIPTION
**Pull request by fishpenguin**
Signed-off-by: fishpenguin <kun.yu@zilliz.com>

/kind bug

issue: #12076

### CODE REVIEW

File: internal/util/rocksmq/server/rocksmq/rocksmq_impl.go [hunk 1]
```diff
@@ -593,7 +593,7 @@ func (rmq *rocksmq) Consume(topicName string, groupName string, n int) ([]Consum
 
 	metaKey := constructCurrentID(topicName, groupName)
 	currentID, err := rmq.kv.Load(metaKey)
-	if err != nil {
+	if err != nil || currentID == "" {
 		log.Debug("RocksMQ: load " + metaKey + " failed")
 		return nil, err
 	}
```
Comment (congqixia) [inline internal/util/rocksmq/server/rocksmq/rocksmq_impl.go:599]: When `currentID` is empty string, the returned value is `nil, nil` which is not recommended.

Comment (fishpenguin) [reply internal/util/rocksmq/server/rocksmq/rocksmq_impl.go:599]: I'll fix this in next pr.

Comment (scsven) [reply internal/util/rocksmq/server/rocksmq/rocksmq_impl.go:599]: btw, the log level in line 597 change to WARN may be better.

Comment (xiaofan-luan) [reply internal/util/rocksmq/server/rocksmq/rocksmq_impl.go:599]: I would suggest to return err if currentID not exist

**Pull request merged**